apollo-router/src/services/router.rs

```diff
@@ -30,9 +31,11 @@ use crate::Context;
 pub type BoxService = tower::util::BoxService<Request, Response, BoxError>;
 pub type BoxCloneService = tower::util::BoxCloneService<Request, Response, BoxError>;
 pub type ServiceResult = Result<Response, BoxError>;
+//#[deprecated]
```

Comment:
If we uncomment this, we get ~164 warnings.

I think we need to resolve all of these before this PR is done.